lib/ui/platform_dispatcher.dart

```diff
@@ -66,6 +66,11 @@ typedef ErrorCallback = bool Function(Object exception, StackTrace stackTrace);
 // A gesture setting value that indicates it has not been set by the engine.
 const double _kUnsetGestureSetting = -1.0;
 
+// The view ID of PlatformDispatcher.implicitView.
+//
+// Keep this in sync with kImplicitViewId in window/platform_configuration.cc.
```

Comment:
While we plan to bake the "implicit view has view ID 0" assumption throughout the engine, the framework and apps should not depend on this. Given this is Dart code and an enterprising user might read this, what do you think of expanding this comment with something like:

```suggestion
// The view ID of PlatformDispatcher.implicitView. This is an
// implementation detail that may change at any time. Apps
// should always use PlatformDispatcher.implicitView to determine
// the current implicit view, if any.
//
// Keep this in sync with kImplicitViewId in window/platform_configuration.cc.
```